l3build.dtx

```diff
@@ -335,12 +335,13 @@
 % The \texttt{build.lua} file used to control \pkg{l3build} is a simple Lua file
 % which is read during execution. In the current release of \pkg{l3build},
 % \texttt{build.lua} is read automatically and can access all of the global
-% functions provided by the script. Thus it may contain a simple list of
+% functions and variables provided by the script. Thus it may contain a simple list of
 % variable settings \emph{or} additional code to customize the build
 % process.
 %
 %The example scripts given in Section~\vref{sec:examples} largely cover the required knowledge in Lua programing.
 % For a more advanced usage, one may consult general Lua documentations including \url{https://www.lua.org/manual/5.3/manual.html} and for the few |texlua| specific additions see section 4.2 of the \LuaTeX{} manual available locally with |texdoc luatex| command line or at \url{https://www.pragma-ade.com/general/manuals/luatex.pdf}.
+% Note that |os.execute| is at version 5.1, which is undocumented there.
```

Comment:
I don't follow the need for this - LuaTeX includes Lua 5.3.

Reply:
Except for `os.execute` that is not at version 5.3 but 5.1.
See [https://tug.org/pipermail/luatex/2015-November/005536.htm](https://tug.org/pipermail/luatex/2015-November/005536.html)